test/support/assertions.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+#
+# Copied `assert_block` method from test-unit project
+#
+# The License is https://github.com/test-unit/test-unit/blob/master/BSDL
+#
```

Comment:
I'm not sure if this is enough for requirements in License of test-unit.
I'd like some advices